experiments/AMIP/moist_mpi_earth/bucket/bucket_init.jl

```diff
@@ -144,13 +149,18 @@ function bucket_init(
     end
 
     earth_param_set = create_lsm_parameters(FT)
-    function α_soil(coordinate_point)
-        (; lat, long) = coordinate_point
-        return typeof(lat)(0.4)
-    end
 
     α_snow = FT(0.8) # snow albedo
-    albedo = BulkAlbedo{FT}(α_snow, α_soil)
+    if albedo_from_file # Read in albedo from data file (default)
+        albedo = BulkAlbedoMap{FT}(regrid_dirpath, α_snow = α_snow, comms = comms_ctx)
+    else # Use spatially-varying function for surface albedo
+        function α_sfc(coordinate_point)
```

Comment:
one question! If the prescribed albedo function is constant, why is it varying so much in the plot? Are the non 0.4 values due to the ocean?

With snow on the surface, we do a linear interpolation between the albedo of ground and albedo of snow, which could cause variation, but I believe there is no snow being modeled yet (P_snow =0), and this would only increase the albedo from the base value of 0.4.

Reply:
Instead of plotting SW albedo vs index, perhaps we could plot it in terms of lat/lon?